### DISCUSSION TITLE
Audio scrubbing in beatmap editor

### DISCUSSION THREAD

**Discussion by sportshead** [ppy/osu#13175] [Ideas]
**Describe the new feature:** 
As a user who wants to create beatmaps on osu!lazer, being able to listen to the audio while scrubbing would be very helpful. I'm imagining something like Blender's audio scrubbing feature.

Basically when you scroll in the editor, it moves the tracker thing (see screenshot), but currently while it it moving you can't hear the audio.
![image](https://user-images.githubusercontent.com/32637656/103266926-97ed8500-49eb-11eb-98c3-44465ae20459.png)

So for example in this song, if you scroll and move the tracker thing to another position (see screenshot 2) it would play the part of the song that says "world this" while you are moving the tracker thing.

![screenshot 2](https://user-images.githubusercontent.com/32637656/103267139-16e2bd80-49ec-11eb-84e8-a830dac06f15.png)

I hope this was clear enough, sorry for my bad photo editing skills

**Comment by bdach** [comment]
Do you mean playback while dragging the tracker? I'm pretty sure that was disabled explicitly due to sounding really strange.

**Comment by sportshead** [comment]
Yes, that's what I mean. Perhaps an option (disabled by default)? Being able to listen while scrolling through the timeline would be helpful too

**Comment by bdach** [comment]
As usual I will push back against this being an option. We have too many already and 95% of users won't probably use half of them.

I was thinking about #10463 when I said that "this might have been disabled". I no longer believe that's the case, so I guess this proposal doesn't counteract any prior statements and can remain open.
